### PR TITLE
oboe: add general purpose getSdkVersion() query

### PR DESCRIPTION
**Pull request by philburk**
Use this for deciding whether to support AAudio.

### CODE REVIEW

File: src/aaudio/AudioStreamAAudio.cpp [hunk 1]
```diff
@@ -83,24 +83,14 @@ AudioStreamAAudio::AudioStreamAAudio(const AudioStreamBuilder &builder)
...
-  if (__system_property_get("ro.build.version.sdk", sdk) != 0) {
-      return atoi(sdk) >= 27;  // SDK Level 27 is Android Oreo 8.1
-  }
-#endif
-  return false;
-}
-
 bool AudioStreamAAudio::isSupported() {
-    if (!isRunningOnAndroid8_1OrHigher()) {
+    const int SDK_8_1 = 27; // OC-MR1
```
Comment (mnaganov) [inline src/aaudio/AudioStreamAAudio.cpp]: Please rename to "ANDROID_8_1". People usually associate "SDK" with API versions, so "SDK 8.1" looks strange.

Comment (philburk) [reply src/aaudio/AudioStreamAAudio.cpp]: Done

**Comment by philburk** [comment]
@dturner - Should this be getAndroidApiVersion() instead of getSdkVersion() ?

**Comment by mnaganov** [comment]
In android.os.Build, this number is called "The SDK version of the software currently running on this hardware device." They also call it "API level" (see https://developer.android.com/guide/topics/manifest/uses-sdk-element.html for an example). So if you don't like 'getSdkVersion', the correct alternative is 'getAndroidApiLevel'.

**Pull request merged**